mne/preprocessing/ica.py

```diff
@@ -556,6 +563,9 @@ def _fit_epochs(self, epochs, picks, decim, verbose):
 
         self.n_samples_ = np.prod(data[:, 0, :].shape)
 
+        # adding reject parameter
+        self.reject_ = epochs.reject
```

Comment:
Perhaps this needs some verbosity to be less magic.

Reply:
to remove. We agreed it's too much magic